spec/requests/v0/form1010_ezrs_spec.rb

```diff
@@ -133,17 +133,70 @@
       )
```

Comment:
This is only at 70% code coverage. Please update spec file to surpass 90% code coverage.

Reply:
The spec/requests/v0/health_care_applications_spec.rb is only at 78.08 % coverage. Please update spec file to surpass 90% code coverage.